extensions/interactions/base.py

```diff
@@ -186,6 +186,11 @@ def rules_dict(self):
 
     @property
     def _rule_description_strings(self):
+        """Returns a dict of rule names to rule descriptions.
```

Comment:
maybe better to say: Returns a dict, where the keys are rule names, and the values are the corresponding rule descriptions.

Reply:
Done